server/schedulers/utils.go

```diff
@@ -63,7 +65,11 @@ func scheduleTransferLeader(cluster schedule.Cluster, schedulerName string, s sc
 			region, peer = scheduleRemoveLeader(cluster, schedulerName, mostLeaderStore.GetId(), s)
 		}
 	}
-
+	if region == nil {
+		log.Debugf("[%v] select no region", schedulerName)
+	} else {
+		log.Debugf("[region %v][%v] select %v to be new leader", schedulerName, region.GetId(), peer)
```

Comment:
I guess the parameter does not correspond.

Reply:
oh, I mistake the order